content/es/docs/reference/glossary/kube-apiserver.md

```diff
@@ -0,0 +1,25 @@
+---
+title: API server
```

Comment:
```suggestion
title: Servidor API
```

(the alternative title `kube-apiserver` remains valid of course)

Reply:
Localised in https://github.com/kubernetes/website/pull/22259/commits/e423bf22516ddde02c86ee8d5e4e2f5c1c241930